Tests/MapboxMapsTests/Annotations/ViewAnnotationManagerTests.swift

```diff
@@ -288,6 +288,22 @@ final class ViewAnnotationManagerTests: XCTestCase {
         XCTAssertEqual(annotationView.frame, CGRect(x: 150.0, y: 200.0, width: 100.0, height: 50.0))
     }
 
+    func testAnnotationPlacementZOrder() {
+        let annotationViewA = addTestAnnotationView(id: "test-id")
+        let annotationViewB = addTestAnnotationView(id: "test-id2")
+
+        // Item closest to map should be the last in the annotation array
+        XCTAssertEqual(mapboxMap.addViewAnnotationStub.invocations.last?.parameters.id, "test-id2")
+
+        // After updating, the item clostst to the map is now furthest from the map
+        try? manager.update(annotationViewA, options: ViewAnnotationOptions(selected: true))
```

Comment:
this function will not call to `mapboxMap.addViewAnnotation(withId:options:)`, therefore the following assertion does not make sense, the stubbed invocation is made in the previous step of this test case and has not been reset, so, with or without this line the test would always succeed